src/main/java/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate.java

```diff
@@ -307,6 +315,27 @@ public DescriptorImpl() {
         load();
       }
     }
+
+    /**
```

Comment:
Can you help me understand how this relates to the template command? I'm failing to make the connection

Reply:
That was a left over from the old diff.